apps/web/src/server/views/applications/case-documentation/documentation-folder.njk

```diff
@@ -61,10 +61,11 @@
 					<hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
 					{% if documentationFiles |length %}
 						<form class="pins-files-list" method="post">
+							<caption class="display--sr-only">Manage Documents in this location</caption>
```

Comment:
this happens on both chrome and firefox. apparently the browser removes the tag caption which is supposed to exist only inside a <table>

![Uploading immagine.png…]()

Reply:
done